.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+# Update version and changelog, then publish to npm
+
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        id: checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        id: node
+        uses: actions/setup-node@v2
+        env:
+          PLAYER_NPM_PUBLISH: ${{ secrets.PLAYER_NPM_PUBLISH }}
+        with:
+          node-version: '16'
+          cache: 'npm'
+
+      - name: Bump version and update CHANGELOG
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions"
+          npm run release
+
+          npm install
```

Comment:
If this is just to update the package lock, I think you can use `npm install --package-lock-only`.

Reply:
Thank you, I've updated this to only update the package-lock 👍